api_coverage.md

```diff
@@ -0,0 +1,108 @@
+<!--
+  -- Copyright (c) 2020, Arm Limited, All Rights Reserved
+  -- SPDX-License-Identifier: Apache-2.0
+  --
+  -- Licensed under the Apache License, Version 2.0 (the "License"); you may
+  -- not use this file except in compliance with the License.
+  -- You may obtain a copy of the License at
+  --
+  -- http://www.apache.org/licenses/LICENSE-2.0
+  --
+  -- Unless required by applicable law or agreed to in writing, software
+  -- distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  -- See the License for the specific language governing permissions and
+  -- limitations under the License.
+--->
+
+# PSA Cryptography API coverage within Parsec
+
+This page describes the current state of Parsec support for the operations, attributes and options that are defined in the PSA Cryptography API 1.0.0. It covers two different sides of the issue:
```

Comment:
When I read `1.0.0` I was thinking: what if we upgrade our operations to a later version? Would be nice to indicate in this document somewhere the information of the exact PSA Crypto API version that we  currently try to match.

Reply:
Will make it more clear - but also, `1.0.0` *is* the version we're trying to match, right?